packages/strapi-plugin-upload/admin/src/containers/InputModalStepperProvider/reducer.js

```diff
@@ -174,6 +209,19 @@ const reducer = (state, action) =>
         );
         break;
       }
+      case 'SET_FILE_TO_DOWNLOAD_ERROR': {
```

Comment:
Can you add some test for those cases?
SET_FILE_TO_DOWNLOAD_ERROR
ON_CHANGE_URLS_TO_DOWNLOAD
FILE_DOWNLOADED
ADD_URLS_TO_FILES_TO_DOWNLOAD

Reply:
They have already been added.